### PR TITLE
[Workflow] Update MethodMarkingStore

### PR DESCRIPTION
**Pull request by noniagriconomie**
| Q             | A
| ------------- | ---
| Branch?       | 4.3 (related to deprecation of single/multiple marking store)
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no  
| Deprecations? | no 
| Tests pass?   | yes  
| Fixed tickets | .
| License       | MIT
| Doc PR        | already documented

Minor fix

### CODE REVIEW

File: src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
  * This store deals with a "single state" or "multiple state" Marking.
  *
  * "single state" Marking means a subject can be in one and only one state at
- * the same time. Use it with state machine or specific workflow.
+ * the same time. Use it with state machine.
```
Comment (noniagriconomie) [inline src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php]: Either remove
Or need to be more clear on « specific » because it is confusing

Comment (lyrixx) [reply src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php]: :+1:

**Review by lyrixx — COMMENTED** [review]
Except one little comment, this is fine.
Thanks 

(Could you also fix the PR description? (respect the template)

File: src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php [hunk 2]
```diff
@@ -33,6 +33,7 @@ final class MethodMarkingStore implements MarkingStoreInterface
     private $property;
 
     /**
+     * @param bool   $singleState
```
[the remark this answers is not in the document]
Comment (noniagriconomie) [reply src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php]: Can I know why?
There is one and not all?

[the remark this answers is not in the document]
Comment (fabpot) [reply src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php]: We don't add phpdocs that do not add any information that is not already part of the method signature (except if we have an interesting description that adds some information about the param).

[the remark this answers is not in the document]
Comment (noniagriconomie) [reply src/Symfony/Component/Workflow/MarkingStore/MethodMarkingStore.php]: @fabpot @lyrixx understood

**Review by lyrixx — APPROVED** [review]

**Pull request merged**